roles/ansible-runner/tasks/runner.yml

```diff
@@ -33,6 +33,12 @@
   secret_file_path: "{{ ansible_runner_secrets_path }}"
   secret_file_user: "{{ item.user }}"
 
+- name: Disable cron job
+  copy:
```

Comment:
If we are going to support disabling specific runners with a change hoist to set item.disabled: True, then I think reverting that commit should automatically re-enable the runner (as long as the disabled runner isn't the bastion playbook, since that would never run to re-enable the runner).

And if we remove the disable file here when it's not set to disabled, your use case of setting -e ansible_runner_disabled=True from the CLI still works since that would disable the bastion playbook runner, and you would need to manually remove that disable file to re-enable the runners.